examples/frameworks/strands_demo/bedrock_agentcore/scripts/update_nat.py

```
@@ -1,4 +1,5 @@
-# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES.
```

Ruff (0.14.7)

The file is executable but no shebang is present

(EXE002)